cirq-core/cirq/ops/state_preparation_channel.py

```diff
@@ -108,13 +107,21 @@ def _kraus_(self):
         return operator
 
     def __repr__(self) -> str:
-        return f'cirq.StatePreparationChannel({proper_repr(self._state)})'
+        return f'cirq.StatePreparationChannel({proper_repr(self.state)})'
+
+    def __str__(self) -> str:
+        return f'StatePreparationChannel({proper_repr(self.state)})'
```

Comment:
We don't need to print the `repr` of `state` here. We can just print it's `str` to keep things cleaner.
```suggestion
        return f'StatePreparationChannel({self.state})'
```

Also, since we are only using `name` in the circuit diagram, let's update the docstring for `name` in the `__init__` function to specify that the name will be used in circuit diagrams.